test/test_mkldnn.py

```diff
@@ -122,6 +122,34 @@ def test_conv2d(self):
                 self._test_serialization(mkldnn_conv2d, (x.to_mkldnn(),))
                 self._test_tracing(mkldnn_conv2d, (x.to_mkldnn(),))
 
+    def test_conv2d_legacy_jit_model(self):
+        """
+        MKLDNN integration used to serialize models with 5d weight for grouped
+        convolutions, we'd like to preserve this behavior
+        """
+        g = 4
+        conv2d = torch.nn.Conv2d(16, 16, 3, groups=g)
+        conv2d_mkldnn = torch.utils.mkldnn.to_mkldnn(conv2d)
+
+        # contrive legacy conv2d module with a 5-d weight
+        o, i, h, w = conv2d.weight.shape
```

Comment:
nit: assert `o % 5 == 0`?

Reply:
It'd fail in reshape anyway if it's not the case